### DISCUSSION TITLE
Email and Registration not Reflected in UI

### DISCUSSION THREAD

**Discussion by pumacustom0c** [karlomikus/bar-assistant#385] [Q&A]
I set up all of the SMTP settings inside my .env file and it does not seem to be working. Upon registration it does not send a confirmation email and there is no forgot password button. 

```
MAIL_MAILER=smtp
# SMTP hostname
MAIL_HOST=smtp.***.org
# SMTP port
MAIL_PORT=25
# SMTP encryption
MAIL_ENCRYPTION=tls
# SMTP username
MAIL_USERNAME=***@mail.***.com
# SMTP password
MAIL_PASSWORD=***
MAIL_REQUIRE_CONFIRMATION=true
MAIL_CONFIRM_URL="https://bar.***.com/confirmation/[id]/[hash]"
MAIL_RESET_URL="https://bar.***.com/reset-password?token=[token]"
MAIL_FROM_ADDRESS="***@mail.***.com"
MAIL_FROM_NAME="***'s Bar Assistant"
```

Also, I set registration to false in my docker-compose.yaml and it did stop the ability to register by throwing an error when someone tries to sign up but the button to register is still there. Is this the intended functionality? I feel like it should remove that registration button entirely.

**Comment by karlomikus** [comment]
Hello, you also need `MAILS_ENABLED=true` on salt-rim container.
As for not sending the mail, this all looks correct, maybe you can check the logs and see if there is some errors there.

**Reply by pumacustom0c** [reply]
Thanks for the response! I tried putting `MAILS_ENABLED=true` under the environment variables for salt-rim within my docker compose file but that did not enable the forgot password button. Is there a different spot for me to add that line? 

docker-compose.yml: 
```
salt-rim:
  environment:
    - MAILS_ENABLED=true
```
